package.json

```diff
@@ -56,9 +56,9 @@
     "npm": ">= 5.8.0"
   },
   "dependencies": {
+    "@buildit/gravity-ui-web": "^3.0.0-alpha.6",
     "@buildit/job-listings": "^1.1.0",
-    "@buildit/gravity-ui-web": "next",
```

Comment:
shouldn't this remain as `next`? Just the `package-lock.json` will change to a specific alpha.6 version?

Reply:
Ah yes, you're right. Will fix that.